### PR TITLE
User guide: add more information about editing a contact

### LINKED ISSUE #180: [PE-D] Edit command unclear
It is not clear that the edit command can take in multiple m/ inputs

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: jonas-chow/ped#11

### PR DESCRIPTION
**Pull request by tanruiquan**
resolves #180

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -194,18 +194,17 @@ Edits the contact at the specified index in the currently viewed list.
 Format: `edit INDEX [n/NAME] [e/EMAIL] [m/MODULE_CODE LESSON_CODE(S)] [p/PHONE] [h/TELEGRAM_HANDLE] [r/REMARK]`
 
 * At least one of the parameters must be provided
-* Existing values of the parameters specified will be erased and updated to the input values
+* Existing values of the parameters specified will be overwritten
+* You can input multiple module codes, and can have multiple lesson codes for a single module code
 
 <div markdown="span" class="alert alert-primary">:information_source: **Note:**
-To remove an existing phone number/telegram handle/remark from a contact, simply input `p/`/`h/`/`r/` respectively, followed with a whitespace. <br>
-e.g. `edit 1 r/` will remove the existing remark from the first contact.
```
Comment (James-Kua) [inline docs/UserGuide.md:201]: I think this example is needed.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -194,18 +194,17 @@ Edits the contact at the specified index in the currently viewed list.
...
+To remove an existing phone number/telegram handle/remark from a contact, simply input `p/``h/``r/` respectively.
 </div>
 
 Examples:
-* `edit 1 p/91234567 e/ben321@gmail.com`: Edits the phone number and email of the 1st contact to be `91234567` and `ben321@gmail.com` respectively
-* `edit 2 n/Ben m/CS2100 T09 B09`: Edits the name and module of the 2nd contact to be `Ben` and `CS2100 T09 B09` respectively
-* `edit 3 h/@BenWasHere r/Overseas`: Edits the telegram handle and remark of the 3rd contact to be `@BenWasHere` and `Overseas` respectively
-* `edit 4 n/Ben m/CS2100 T09 B09 p/91234567 e/ben321@gmail.com h/@BenWasHere r/Overseas`: All parameters present in one command
+* `edit 1 e/ben321@gmail.com p/91234567`: Edits the email and phone number of the 1st contact to be `ben321@gmail.com` and `91234567`respectively
+* `edit 2 m/CS2100 T09 B09 m/CS2103T T01`: Edits the modules of the 2nd contact to be `[CS2100 T09 B09], [CS2103T T01]`
```
Comment (James-Kua) [inline docs/UserGuide.md:206]: I think the square brackets can be omitted, since they are a result of the UI display.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -194,18 +194,17 @@ Edits the contact at the specified index in the currently viewed list.
 Format: `edit INDEX [n/NAME] [e/EMAIL] [m/MODULE_CODE LESSON_CODE(S)] [p/PHONE] [h/TELEGRAM_HANDLE] [r/REMARK]`
 
 * At least one of the parameters must be provided
-* Existing values of the parameters specified will be erased and updated to the input values
+* Existing values of the parameters specified will be overwritten
```
Comment (James-Kua) [inline docs/UserGuide.md:197]: I do think the deleted line is a more clear as it acts as an explanation for the word "overwritten"

File: docs/UserGuide.md (hunk 1 above)
Comment (tanruiquan) [reply docs/UserGuide.md:201]: @James-Kua I gave the example under the `Examples:` section as i felt like is more appropriate there

File: docs/UserGuide.md (hunk 3 above)
Comment (tanruiquan) [reply docs/UserGuide.md:197]: Fair point, i would revert the changes, thanks!

File: docs/UserGuide.md (hunk 1 above)
Comment (James-Kua) [reply docs/UserGuide.md:201]: I see, didn't catch that, my bad!

**Pull request merged**